src/components/admin/UserProfileModal/components/links/ProfileModalLink.tsx

```diff
@@ -7,16 +7,14 @@ import { ContainerClassName } from "types/utility";
 
 import { useProfileModalLinkIcon } from "utils/profileModalLinkUtilities";
 
-import "./ProfileModalLink.scss";
-
 export const ProfileModalLink: React.FC<
   { link: ProfileLink } & ContainerClassName
 > = ({ link, containerClassName }) => {
   const linkIcon = useProfileModalLinkIcon(link.url);
 
   return (
     <a
-      className={classNames("ProfileModalLink", containerClassName)}
+      className={classNames("ProfileModalLink mr-4", containerClassName)}
```

Comment:
Same here for the `.scss` file classNames.

Reply:
as agreed, moved class name info to `data-bem`:  https://github.com/sparkletown/sparkle/pull/2931/commits/c90bfc174c869b913ac93ae9776b886e624279c8